qiskit/_quantumcircuit.py

```diff
@@ -5,17 +5,37 @@
 # This source code is licensed under the Apache License, Version 2.0 found in
 # the LICENSE.txt file in the root directory of this source tree.
 
+# pylint: disable=cyclic-import
```

Comment:
is this still necessary?

Reply:
Yes, this is needed because pylint is overly conservative and will error even though the code works (there is a potential issue on python < 3.5, but that won't affect us). Since we need to import the circuit unroller backend to make the qasm->circuit translation work. This kind of thing is why I started the discussion in #1179